tests/TestCompleteOrder.py

```diff
@@ -9,37 +9,49 @@ class TestCompleteOrder(unittest.TestCase):
 
     """
     TestCompleteOrder tests the Order is 'complete' with the appropriate attributes.
-
     Method List:
     -test_order_is_complete
-
     Argument List:  unittest.TestCase gives the unittest model knowledge on what to test.
-
     Author: Whitney Cormack, Temporary Oysters
     """
 
     @classmethod
-    def setUp(self):
+    def setUpClass(self):
         """
         Sets up a Test User, "Joey", so we can test if our code is behaving as expected.
         """
-
-        self.joey = customer.Customer('Joey', '787 East Silver St', 'Lebanon', 'Ohio', '35622', '5551231234', 'j@j')
-        self.joey_mastercard = payment_options.PaymentType("Mastercard", "acct1234", 1)
-        self.joeys_cart = shoppingcart.ShoppingCart("joey")
+        self.joe = customer.Customer('Joe llama', '787 Mulberry St', 'Nashville', 'TN', '35622', '5551231234', 'jllama@j.com')
+        self.joe.register_customer(self.joe)
+        self.joe.set_status_to_active()
+        self.joe_mastercard = payment_options.PaymentType("Mastercard", "1234", 5)
+        self.joe_mastercard.add_payment_type()
+        self.joes_cart = shoppingcart.ShoppingCart("joe")
         self.puppies = products.Product("puppies", 3)
-        self.joeys_cart.add_to_cart(self.puppies)
-        self.joeysorder = complete_order.Order(self.joey, self.joey_mastercard)
+        self.joes_cart.add_to_cart(self.puppies)
+        self.joes_order = complete_order.Order()
+
 
     def test_order_is_complete(self):
         """
         Tests that our user has created an order which includes correct attributes with assigned values.
         An Order is 'complete' when it has attributes and values for a customer, a payment type, products in cart, and cart total cost.
         """
-        self.assertIsInstance(self.joeysorder, complete_order.Order)
 
-    def test_can_get_order_status(self):
-        self.assertFalse(self.joeysorder.get_order_status())
+        self.assertIsInstance(self.joes_order, complete_order.Order)
+
+    def test_order_has_proper_attributes(self):
+        """
+        Tests order is able to add correct CUSTOMER & PAYMENT TYPE attributes.
+        """
+        self.assertIsNotNone(self.joes_order.customer)
+        print("HEYYYY LOOK AT ME!", self.joes_order.payment_type)
```

Comment:
Look at your tests. I believe this test passes because it has the option to return False in the if/else statement in `Orders.retrieve_payment_type_from_db`
See the comment in that file and let me know what your think